test/src/conv.rs

```diff
@@ -103,3 +100,26 @@ pub unsafe fn deep_clone(a: ocaml::Value) -> ocaml::Value {
 pub fn pair_vec() -> ocaml::Value {
     vec![("foo", 1), ("bar", 2isize)].to_value()
 }
+
+#[ocaml::func]
```

Comment:
I think this should go in `test/src/types.rs` and the OCaml tests should go in `test/src/types.ml`